packages/babel-preset-gatsby/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "babel-preset-gatsby",
+  "version": "2.5.0",
+  "author": "Philipp Spiess <hello@philippspiess.com>",
+  "devDependencies": {
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0"
+  },
+  "license": "MIT",
+  "main": "src/index.js"
```

Comment:
If we don't have/need a compile step, this should probably change! Otherwise we should compile and this should be dist/index.js or something! See a lot of the other gatsby packages for more detail!

Reply:
Do you think we need babel here? I'm not sure. Right now it works just fine without but I'm also on Node 9.7 - some developers might run an older Node maybe?

Reply:
I think we need to ensure it runs on the oldest Node LTS version (i.e. [6.x until April, 8.x until next December](https://github.com/nodejs/Release)), so probably this needs to be transpiled.